Runtime/Scripts/MobiledgeXIntegration.cs

```diff
@@ -357,20 +269,8 @@ public async Task ConfigureMobiledgeXSettings(bool LocationNeeded = true)
             }
             tcpPort = (int)settings.TCP_Port;
             udpPort = (int)settings.UDP_Port;
-            // Checking if edge is enabled on the device used or not
-            if (!IsEdgeEnabled())
-            {
-                return;
-            }
-            // Getting location from the device, needed for FindCloudlet and VerifyLocation
-            // Location is ephemeral, so retrieve a new location from the platform. May return 0,0 which is
-            // technically valid, though less likely real, as of writing.
-
-            if (LocationNeeded)
-            {
-                location = await GetLocationFromDevice();
-            }
-
+            // Get Location From Device
+            location = await GetLocationFromDevice();
```

Comment:
Location should be just before FindCloudlet, if actually empty, FindCloudlet shouldn't even be called as it's a required parameter. Complain loudly, and often if it's a developer task.

Reply:
Fixed